app/assets/javascripts/validation.coffee

```diff
@@ -349,6 +347,13 @@ $(document).ready ->
               params: {}
             errors.push newError
 
+  validateLatLonRanges = (errors) ->
+    if $("[id^=variable_draft_draft_characteristics_index_ranges_lat_range_]").length > 0
```

Comment:
It looks like all you are doing is visiting the field if the parent id exists on the page. So those fields will be visited every time the page is validated, right? We usually only validate a field if it has already been visited, but here it seems the field will be marked as visited even if the user hasn't visited it.

Reply:
Added fix in method validateLatLonRanges